examples/experimental/9p.yaml

```diff
@@ -1,3 +1,11 @@
+# ⚠️ ⚠️ ⚠️ `template://experimental/9p` will be removed in in Lima v1.0,
+# as 9p will graduate from experimental and will be the default mount type for QEMU.
+#
+# Use the following command instead:
+# ```
+# limactl create --vm-type=qemu --mount-type=9p template://default
```

Comment:
It just said that `9p` is the default for `qemu`, so

```suggestion
# limactl create --vm-type=qemu template://default
```

Reply:
fixed